src/renderer/components/commands-editors.tsx

```diff
@@ -82,6 +82,14 @@ export class EditorDropdown extends React.Component<EditorDropdownProps, EditorD
       );
     }
 
+    result.push(
+      <Button
+        icon='grid-view'
+        text='Reset Layout'
+        onClick={appState.resetEditorLayout}
+      />
+    );
```

Comment:
This button is very stood out from the content of that menu. Here for more beautiful be that if that button also uses `<MenuItem />` as the above, that allows to not stand out from styles. For creating the space between this button and above you can use the `<MenuDivider />` component.

Here example for your case:

```suggestion
    result.push(
      <React.Fragment key={'fragment-reset-layout'}>
        <MenuDivider />
        <MenuItem
          icon='grid-view'
          key='reset-layout'
          text='Reset Layout'
          onClick={appState.resetEditorLayout}
        />
      </React.Fragment>
    );
```

And how it's look in the app:

![Untitled](https://user-images.githubusercontent.com/24681191/80436753-83ab3280-8908-11ea-8385-d9de1cdd430a.png)

@erickzhao Here used `React.Fragment`  instead of short-form because also need to have a unique `key`.

Reply:
Wow, that looks so much better, thank you so much! I've updated to this so the button stands out as something different than one of the Editor buttons, but doesn't look so clunky and out of place. Also used those components recommended and added in the missing key, thanks again!